server/src/config.js

```diff
@@ -57,6 +57,13 @@ var conf = convict({
     arg: "s3BucketName"
   },
   fxa: {
+    server: {
+      doc: "The FxA server base URL",
+      format: String,
+      default: "https://stable.dev.lcip.org",
+      env: "FXA_SERVER",
```

Comment:
This name is confusing – we're really thinking just about a base URL for profile images. And I notice that this matches the origin of the profileServer below.